### DISCUSSION TITLE
Prevent Escape from Exiting Fullscreen when trying to access the command line in fullscreen

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#2060] [Ideas]
I think it would be useful if when in fullscreen when pressing `Escape`, the command line opens, but when holding down escape, Monkeytype exits fullscreen. IMO it would be easier for users than needing to use an alternate keyboard shortcut to open the command line in fullscreen.

**Comment by Miodec** [comment]
Use `ctrl + shift + p` when in full screen.

**Reply by unrealapex** [reply]
@Miodec Chrome has an experimental feature for this: [`navigator.keyboard.lock()`](https://developer.mozilla.org/en-US/docs/Web/API/Keyboard/lock). I know that this would only work for Chrome but would it be worth implementing? Also, just FYI, `ctrl + shift + p` opens up the print dialogue in Firefox, I didn't take that into account when adding the keyboard shortcut.
